TeleportMenu.lua

```diff
@@ -351,11 +351,11 @@ function tpm:checkQuestCompletion(quest)
 	end
 end
 
-function tpm:CreateFlyout(flyoutId, iconId, yOffset, name)
-	if db.showOnlySeasonalHerosPath and subtype == "path" then
+function tpm:CreateFlyout(flyoutData, yOffset)
```

Comment:
Don't think yOffset is used anymore now that I changed the reversal to use xOffset without recreating the buttons.

```suggestion
function tpm:CreateFlyout(flyoutData)
```